src/components/map/components/Vehicles.tsx

```diff
@@ -27,7 +27,10 @@ export const Vehicles = ({mapCameraRef, vehicles, onPress}: Props) => {
       ref={shapeSource}
       shape={vehicles}
       tolerance={0}
+      buffer={0}
```

Comment:
What happens if `buffer` and `tolerance` are removed?

Reply:
`tolerance` definitely had an effect when I tested it. Less objects disappeared. I'm not sure about `buffer,` though. I can investigate a bit.

Reply:
Removed the `buffer` option. Performance seems to be sufficient with default buffer value.